package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@vtex/clients",
-  "version": "2.8.0",
+  "version": "2.9.0-beta.0",
```

Comment:
I believe we can keep as it was before, since we'll use releasy to bump the version, create the tag and release the package version